docs/modules/hdfs/pages/usage-guide/operations/pod_disruptions.adoc

```diff
@@ -0,0 +1,108 @@
+
+= Pod disruptions
```

Comment:
Shouldn't this be Pod disruption budget?

Reply:
I would consider this a implementation detail. In the future we might want something more smart than PDBs, e.g. k8s asking the hdfs-operator "can i safely evict this pod?".
This would still count as Pod disruptions, but not PDB.
Same as we call the concept guide on affinities "Pod placement".

Reply:
Yes and no.
"configure pod disruptions" sounds bad.

@adwk67 @Jimvin calling for a native speaker. I understand what Sebastian means. What do you think?

Reply:
The following link might give some inspiration :)
https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#pod-disruption-conditions

Reply:
I only commented here. I understand why you want to keep the title like this, but...

Reply:
If we refer to "controlling pod disruptions" I think it is OK. That makes it a little more clear that whatever we configure is aimed at restraining disruptions rather than inviting them!

Reply:
Just to be sure: And heave that as the heading on the global concepts pages as well as in every operator?
@lfrancke does that work for you as well?

Reply:
That are the current concepts:
Honestly feels a bit weird that one of them will have a "Configure xxx" prefix, but I don't care too much ;)
![image](https://github.com/stackabletech/hdfs-operator/assets/29303194/0b21d7a4-55f1-4710-b6ba-44c64c2c2400)

Reply:
"Configure ..." is something you would use for a guide. If you are explaining a concept, the concept should be the title of the page. But I would argue that the concept is more something like "Service stability" or "Reliability" (which can be affected by pod disruptions). The concept pages should explain things from taking a step back and looking at the big picture, contextualize. Here is something about it in the diataxis framework: https://diataxis.fr/explanation/#writing-good-explanation

Reply:
Would be great if we could reach a decision, so that I don't need to update 15 similar PRs later on :)
What would be your favorite?

Just as a reminder: What we are documenting here is what *this* operator is doing in terms of pod disruptions, e.g.  documenting how the PDBs are calculated. The concepts page is added in https://github.com/stackabletech/documentation/pull/454

Reply:
Keep it as it is